src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -1,34 +1,42 @@
 package edu.kis.vh.nursery;
 
-public class defaultCountingOutRhymer {
-
-    private int[] NUMBERS = new int[12];
-
-    public int total = -1;
-
+public class DefaultCountingOutRhymer {
+    private static final int SIZE = 12;
+    private static final int ERR_CODE = -1;
+    private static final int FULL_INDEX = SIZE - 1;
+    
+    private int[] numbers = new int[SIZE];
+    private int total = ERR_CODE;
+    
     public void countIn(int in) {
-        if (!isFull())
-            NUMBERS[++total] = in;
+        if (!isFull()) {
+            numbers[++total] = in;
+        }
     }
-
-        public boolean callCheck() {
-            return total == -1;
+    
+    public boolean callCheck() {
+        return total == ERR_CODE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.